src/recent-fare-contracts/RecentFareContract.tsx

```diff
@@ -145,100 +142,58 @@ export const RecentFareContract = ({
       }
       style={{minWidth: width * 0.6}}
     >
-      <View style={styles.travelModeWrapper}>
-        <TransportModes
-          iconSize="xSmall"
-          modes={fareProductTypeConfig.transportModes}
-          style={{flex: 2}}
-        />
-      </View>
-
       <View style={styles.productName} testID={testID + 'Title'}>
         <ThemeText
-          typography="body__secondary--bold"
+          typography="body__primary--bold"
           color={interactiveColor.default}
         >
-          {getReferenceDataName(preassignedFareProduct, language)}
+          {getTextForLanguage(fareProductTypeConfig.name, language) ?? ''}
         </ThemeText>
       </View>
 
-      {direction !== undefined && pointToPointValidity && (
-        <View style={styles.harbors}>
-          <FareContractFromTo
-            backgroundColor={theme.color.background.neutral['0']}
-            mode="small"
-            rfc={recentFareContract}
+      <View style={styles.recentFareContractDetailItems}>
+        <FareContractFromTo
+          rfc={recentFareContract}
+          backgroundColor={theme.color.background.neutral[0]}
+          mode="small"
+        />
+
+        {userProfilesWithCount.map(() => (
+          <FareContractDetailItem
+            content={[
+              `${getTransportModeText(
+                fareProductTypeConfig.transportModes,
+                t,
+              )}`,
```

Comment:
Something a little strange here, iterating over user profiles to show transport modes. By reading this I would expect that the transport modes are shown multiple times if there are multiple user profiles.